### ISSUE TITLE
initialMessages in useEdgeRuntime  ignored

### ISSUE THREAD

**Issue by igo** [Yonom/assistant-ui#1089]
Hi, I am trying to load inital messages but they seem to be ignored.  I see just an empty chat.

```tsx
  const initialMessages: CoreMessage[] = [
    { "role": "system", "content": [{ "type": "text", "text": "You are weather assistant" }] },
    { "role": "user", "content": [{ "type": "text", "text": "What's the weather in Paris?" }] },
    { "role": "assistant", "content": [{ "type": "text", "text": "It's sunny." }] },
  ];

  const runtime = useEdgeRuntime({
    api: "http://localhost:4000/chat",
    initialMessages: initialMessages,
  });

<Thread runtime={runtime} />
```
<img width="754" alt="image" src="https://github.com/user-attachments/assets/7695a57a-5622-4bcf-b196-b70f70c239c6">

**Comment by Yonom** [comment]
fixed in @assistant-ui/react (0.5.96)

**Comment by Yonom** [comment]
thanks for reporting
